matter_server/server/client_handler.py

```diff
@@ -229,7 +231,7 @@ def _send_message(self, message: MessageType) -> None:
         try:
             self._to_write.put_nowait(_message)
         except asyncio.QueueFull:
-            self._logger.error(
+            self._logger.exception(
```

Comment:
Which rule is this?

Reply:
I don't know - its very hard to track

Reply:
I think it's this one:
https://docs.astral.sh/ruff/rules/error-instead-of-exception/

That's very subjective I'd say. Not sure that we should follow that one all the time.

Reply:
I've disabled this rule in the config